src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1251,6 +1251,15 @@ internal static bool PathIsUnc(string path, bool networkOnly = false)
 #endif
         }
 
+        internal static bool PathIsDevicePath(string path, bool pathIsUnescaped)
+        {
+#if UNIX
+            return false;
+#else
+            return path.StartsWith(@"\\.\") || path.StartsWith(@"\\?\") || (pathIsUnescaped && path.StartsWith(@"\\`?\"));
```

Comment:
You shouldn't explicitly check for escaped characters.  By the time the path is in the FileSystemProvider, the path should be unescaped.  So if you see the backtick, it should be treated as literal.  In your test below, if the path contains a wildcard character, you should change the test to use literalpath or escape it.

Reply:
I wouldn't call it a bug. When looking at the implementation it's clearly working exactly as the author(s) designed, it's not just some small oversight or a coding mistake. Regardless of how we classify it, I feel like it's out of scope to fix that in this PR.

Reply:
@MartinGC94 it seems I don't have permission to commit to your branch, so you can see my proposed changes here: https://github.com/PowerShell/PowerShell/compare/master...SteveL-MSFT:PowerShell:fixdevicepath?expand=1

Reply:
Weird, I would have assumed this: 
![image](https://user-images.githubusercontent.com/42123497/236896575-02a6b348-0d37-4a19-a401-cece46b0fd35.png)
would be enough.  
I couldn't figure out how to merge those changes into my branch so I just copied them manually.